typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "<%= moduleName %>",
+	"version": "0.0.0",
+	"description": "",
+	"license": "MIT",
+	"repository": "<%= githubUsername %>/<%= moduleName %>",
+	"author": {
+		"name": "<%= name %>",
+		"email": "<%= email %>",
+		"url": "<%= website %>"
+	},
+	"engines": {
+		"node": ">=6"
+	},
+	"scripts": {
+		"build": "del dist && tsc",
+		"prepare": "npm run build",
```

Comment:
Why `prepare`? I usually use `prepublishOnly`.

Reply:
> Prepare as build script in `package.json`. [npm-scripts](https://docs.npmjs.com/misc/scripts) describes `prepare`, `prepack` and `prepublishOnly`. I personally like to check package content via `npm pack` so `prepublishOnly` is not helpful here. `prepare` also runs on a `npm install` without arguments as its done after a `git clone`. This might save some time as it hasn't to be done manually but could be annoying on WIP feature branches. The safest solution would be `prepack` but I think `prepare` is a nice to have.

Reply:
I changed to `prepack` as `prepare` runs into errors with a fresh start repo. `prepare` is not needed anyway so use the only `prepack` as it runs on `npm pack` and `npm publish`.